Cargo.toml

```diff
@@ -47,20 +50,16 @@ codegen-units = 16
 
 rkv = { git = "https://github.com/holochain/rkv.git", branch = "master" }
 lmdb-rkv = { git = "https://github.com/holochain/lmdb-rs.git" }
-#lmdb-rkv = { path = "../../rust/lmdb-rs" }
-# observability = { path = "../../rust/observability" }
+holochain_wasmer_common = { git = "https://github.com/holochain/holochain-wasmer.git", branch = "bump-serde" }
+holochain_wasmer_guest = { git = "https://github.com/holochain/holochain-wasmer.git", branch = "bump-serde" }
+holochain_wasmer_host = { git = "https://github.com/holochain/holochain-wasmer.git", branch = "bump-serde" }
+holochain_serialized_bytes = { git = "https://github.com/holochain/holochain-serialization.git", branch = "bump-serde" }
```

Comment:
Can we unpatch these now?

Reply:
Need to cut new releases of these crates first

Reply:
here is a PR on serialization to get that moving - https://github.com/holochain/holochain-serialization/pull/39

Reply:
same here - https://github.com/holochain/holochain-wasmer/pull/53 - i can't approve these PRs because i opened them... even though it's your branch @maackle :sweat_smile: